modules/timer/index.js

```diff
@@ -80,6 +80,10 @@ export class Timer extends React.Component<Props, State> {
 		this.setState(() => ({loading: false}))
 	}
 
+	refresh = () => {
+		this._refresh()
+	}
+
```

Comment:
This seems somewhat kludgey to me. Is there a particular reason why we need a shim?

Reply:
RN’s flow types require that functions passed to RefreshControl’s refresh prop be of the type `() => void`, and all Async functions return Promise<>, which causes type errors at each call site. 

My other alternative here is to typecast this.refresh as a variable in the render method here, but that requires that we cast through `any`, so I almost prefer this way. 

Idk.

Reply:
```js
refresh = () => {
	let start = Date.now()
	this.setState(() => ({loading: true}))

	this.updateTime()

	let tick = Promise.resolve(null)

	let elapsed = Date.now() - start
	if (elapsed < 500) {
		tick.then(() => delay(500 - elapsed))
	}

	tick.then(() => {
		this.setState(() => ({loading: false}))
	})
}
```

That's my other option for this method lol

Reply:
Alright, just curious.